resources/js/chat/conversation-list.tsx

```diff
@@ -21,14 +22,20 @@ function renderChannels(type: SupportedChannelType) {
   if (channels.length > 0 || type === 'ANNOUNCE' && core.dataStore.chatState.canChatAnnounce) {
     const title = trans(`chat.channels.list.title.${type}`);
 
+    const chunks = chunk(channels, 100);
+
     return (
       <React.Fragment key={type}>
         <div className='chat-conversation-list__group'>
           <div className='chat-conversation-list__header'>
             <span className='chat-conversation-list__header-text'>{title}</span>
             <span className='chat-conversation-list__header-icon' title={title}><i className={icons[type]} /></span>
           </div>
-          {channels.map((channel) => <ConversationListItem key={channel.channelId} channel={channel} />)}
+          {chunks.map((c, index) => (
```

Comment:
Might be worth adding a comment somewhere in here mentioning that this is an optimisation.